### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by Habtamu210**
- Todo list with add and remove features

### CODE REVIEW

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Habtamu210, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a README file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,111 +1,213 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](example.com)
+> Add a link to your deployed project.
+
```
Comment (TadesseDev) [inline README.md]: - Dear @Habtamu210, this lines are a suggestion/direction and should not be part of your readme. Please concider removing/replacing all.

**Review by TadesseDev — COMMENTED** [review]
### Hi @Habtamu210 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑🛑
yPkfDFf

- [ ] Please consider removing non-important contents from your readme, The template is designed by Microverse for us all to use to document our projects professionally and has it filled with content related to each project we may be working on. currently, I can see you have not removed the guiding contents  which start with `>` and `directions set by Microverser`

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by topeogunleye — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @Habtamu210,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟

wiserhood-cheers

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good summary description :heavy_check_mark:

## [Optional] suggestions:
 * Nothing to mention.

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @topeogunleye in your question so I can receive the notification

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**